### PR TITLE
osu!tourney

### PR DESCRIPTION
**Pull request by MegaApplePi**
Fixes:
- [x] ASG compliant
  - lowered headings
  - made new headings
- reworded a many sentences
- [x] removed notes about skinning video/`.mp4` (#145)

### CODE REVIEW

File: wiki/osu!tourney/en.md [hunk 1]
```diff
@@ -63,36 +76,52 @@ BufferTimeoutTime = 20000
...
--   **BufferDangerTime:** The remaining time in the replay buffer before the client pauses to wait for more replay frames. It is not recommended to change this.
--   **RankingChatDelay:** Delay (in milliseconds) before the scores hide and show the chat again on the ranking screen.
--   **Height:** Height of your stream. osu!tourney will calculate the Width automatically so it fits 16:9. The black control panel is not included in that height - so you actually need to have a 1440p screen to stream your tournament in 1080p. Default: 720, allowed range: 568-1440
+- `TeamSize` Define the size of your teams here. Depending on this, osu!tourney either opens 1-8 windows per team. Maximum value is 8.
+- `ClientNameSize` Change the yellow player names size.
+- `privateserver` Leave this untouched. It does not matter for you.
+- `acronym` Put an abbreviation for your tournament here, e.g. 'OWC 2013'.
+  More details at **[Multiplayer session creation and handling](Osu!tourney#Multiplayer_session_creation_and_handling "wikilink")**.
```
Comment (RaikireHiuduo) [inline wiki/osu!tourney/en.md]: Section anchor, own page.

No idea.

Comment (MegaApplePi) [inline wiki/osu!tourney/en.md]: whoops, forgot to remove this

**Comment by MegaApplePi** [comment]
section linking is kind of a ¯\_(ツ)_/¯ because cl8n made a few adjustments to fix the section heading IDs... they seem to be working now but I would hold off on them since they are probably subject to change

File: wiki/osu!tourney/en.md [hunk 2]
```diff
@@ -1,56 +1,69 @@
-**osu!tourney** is the designated multi-spectator client used to view and stream entire online multiplayer matches at once.
+# osu!tourney
 
-The access is limited to tournament hosts and their staff. Tournament hosts have to contact **tournaments@ppy.sh** to apply for access, which is limited for the time needed to conclude the tournament they host.
-Using the client is not meant for public right now and requires approval from the game administration.
+**osu!tourney** is the designated multi-spectator client used to view and stream entire online multiplayer matches at once.
```
Comment (sr229) [inline wiki/osu!tourney/en.md]: L3: Revise. Suggested ``**osu!tourney** is the official multi-spectator client used to view and stream entire multiplayer matches at once in osu!.``

File: wiki/osu!tourney/en.md [hunk 3]
```diff
@@ -1,56 +1,69 @@
...
 
-**Any guarantee about its functionality is excluded if you use it for a different purpose than streaming tournaments or multiplayer event sessions. The client is not for private use.**
+Access to the osu!tourney client is limited to tournament hosts and their staff.
```
Comment (sr229) [inline wiki/osu!tourney/en.md]: L5 : Revise. Suggested ``The access to osu!tourney client is limited to a specific tournament's hosts and their staff.``

File: wiki/osu!tourney/en.md [hunk 4]
```diff
@@ -1,56 +1,69 @@
...
+The number of windows opened cannot currently be changed, but this behaviour will be modified in the future.
 
-After you have completed organizing your event, send in an e-mail to **tournaments@ppy.sh** including:
+Keep in mind that in order to use osu!tourney, you must be part of the [Tournament Staff](https://osu.ppy.sh/g/26) usergroup; otherwise, you won’t be able to login properly to it.
 
-1.  When does the tournament start? When does it end?
-2.  Any link to follow the tournament which is publically accessible
-3.  an exhaustive list of concrete osu! usernames that want to use osu!tourney
-4.  Anything else of importance
+### How do I get access?
```
Comment (sr229) [inline wiki/osu!tourney/en.md]: L26: Revise header to passive. Suggested ``### Getting access to the client.``

**Pull request merged**